phpunit.xml.dist

```diff
@@ -8,6 +8,9 @@
   <filter>
     <whitelist>
       <directory suffix=".php">src</directory>
+      <exclude>
+        <directory suffix=".php">src/Logging/V2</directory>
```

Comment:
That definitely seems like the most robust logic - does it also handle the `v1beta2` case?